L1Trigger/TrackFindingTracklet/src/FitTrack.cc

```diff
@@ -1005,7 +1004,11 @@ void FitTrack::execute(const ChannelAssignment* channelAssignment,
       countFit++;
 
 #ifdef USEHYBRID
-      trackFitKF(bestTracklet, trackstublist, stubidslist);
+      if (settings_.fakefit()) {
+        trackFitFake(bestTracklet, trackstublist, stubidslist);
+      } else {
+        trackFitKF(bestTracklet, trackstublist, stubidslist);
```

Comment:
by multiple stubs per layer, do you mean the lines of code defined in trackFitKF(,,) that is by passed by these changes ?
If so, I already added this change when plotting main&&noDR

Reply:
No that was to not have the old KF fit to run in the back. I mean setting Settings::doMultipleMatches_ to false.